plasmapy/formulary/tests/test_ionization.py

```diff
@@ -39,3 +44,19 @@ def test_Saha():
 
     with pytest.raises(u.UnitTypeError):
         Saha(g_j, g_k, 1e19 * u.kg, E_jk, T_e)
+
+
+def test_thomas_fermi_ionization_state():
+    n = 1e29 * u.m ** -3
+    T_e = 11604518.12155008 * u.K
+    z = 6
+
+    val = thomas_fermi_ionization_state(z, n, T_e)
+    assert val.unit == u.dimensionless_unscaled
+    assert np.isclose(val.value, 5.849715866306117)
+
+    with pytest.warns(u.UnitsWarning):
+        thomas_fermi_ionization_state(z, 1e29, T_e)
+
+    with pytest.raises(u.UnitTypeError):
+        thomas_fermi_ionization_state(z, 1e29 * u.kg, T_e)
```

Comment:
Perhaps split this test up into multiple smaller tests?  Currently if one of the `assert` statements raises an exception, the tests of the warning and exception won't be run.

Reply:
Is there a specific rule for naming multiple tests of the same function?

Reply:
It's generally helpful to end test names with a description of what is being tested, like `test_atomic_number_exceptions`.